memory.js

```diff
@@ -0,0 +1,201 @@
+var Game = function(boardElements) {
+	this.boardElements = boardElements;
+	this.tileClass = boardElements.tileHtmlElementClass.substr(1);
+	this.maxValue = boardElements.height*boardElements.width/2;
```

Comment:
I wouldn't be afraid of whitespace within lines :wink: Makes things more readable.